proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -2484,6 +2485,29 @@ message GetCacheWarmerConfigResponse {
   bool isCacheWarmerEnabled = 2;
 }
 
+message ListRouterCompatibilityVersionsRequest {
+}
+
+message ListRouterCompatibilityVersionsResponse {
+  Response response = 1;
+  repeated int32 versions = 2;
```

Comment:
We should be consistent here. Either go with string everywhere or with int.

Reply:
Ideally it would be `int32` everywhere, but then there were validation issues. So the inputs are strings, and the return values are integers to reflect what the versions actually are.

I can make them all strings if you insist, but I don't think it's meaningful nor helpful.

Reply:
I've changed it to be a string everywhere (including composition). The `v` is also part of the version.